<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorChooser</class>
 <widget class="QWidget" name="ColorChooser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QSlider" name="redSlider">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>22</width>
     <height>160</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="blueSlider">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>22</width>
     <height>160</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="greenSlider">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>22</width>
     <height>160</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSpinBox" name="redBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>48</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="blueBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>170</y>
     <width>48</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="greenBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>170</y>
     <width>48</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>91</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>30</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSlider" name="alphaSlider">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSpinBox" name="alphaBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>200</y>
     <width>48</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QLabel" name="grad">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>170</width>
     <height>160</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Alpha</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
